ui/vic-ui-service/src/main/java/com/vmware/vicui/VicUIServiceImpl.java

```diff
@@ -137,8 +136,17 @@ private ServerInfo getServerInfoObject(String serverGuid) {
 		return null;
 	}
 
+    private void setThumbprint(ServerInfo sinfo) {
+        String thumbprint = sinfo.thumbprint;
+        if (thumbprint != null) {
+            _thumbprints.add(thumbprint.replaceAll(":", "").toLowerCase());
+        }
+        ThumbprintTrustManager.setThumbprints(_thumbprints);
```

Comment:
For our case there will be only one thumbprint (one for VC), but `ThumbprintTrustManager` is made for general use where it could handle more than one certificate, as far as my understanding goes.